src/createServer.js

```diff
@@ -1,3 +1,61 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const toCase = url.searchParams.get('toCase');
+    const PATH = url.pathname.slice(1);
```

Comment:
Consider renaming the `PATH` variable to `path` to follow JavaScript naming conventions, which typically use camelCase for variable names.